doc/api/async_context.md

```diff
@@ -286,6 +293,16 @@ emitter.emit('my-event');
 asyncLocalStorage.getStore(); // Returns the same object
 ```
 
+### `asyncLocalStorage.name`
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* {string|undefined}
```

Comment:
`name` can be an empty string if not provided, and this makes the type of the name consistent.

Reply:
Does the spec allow string coercion too? As in, if I pass `null` would I get `"null"`?

Reply:
Answered my own question. Updated to stringify the input and have it return an empty string.